README.md

```diff
@@ -1,15 +1,17 @@
 # Action for Running MATLAB Tests
+The [Run MATLAB Tests](#run-matlab-tests) action enables you to run MATLAB&reg; and Simulink&reg; tests and generate test and coverage artifacts on a [self-hosted](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) or [GitHub&reg;-hosted](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners) runner:
+- To use a self-hosted runner, you must set up a computer with MATLAB as your self-hosted runner and register the runner with GitHub Actions. The runner uses the topmost MATLAB release on the system path to execute your workflow.
+- To use a GitHub-hosted runner, you must include the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to set up your preferred MATLAB release (R2021a or later) on the runner.
 
-The [Run MATLAB Tests](#run-matlab-tests) action enables you to run MATLAB&reg; and Simulink&reg; tests and generate test and coverage artifacts on a self-hosted or GitHub&reg;-hosted runner. The action uses the topmost MATLAB version on the system path.
 
 ## Examples
 Use the **Run MATLAB Tests** action to automatically run tests authored using the MATLAB unit testing framework or Simulink Test&trade;. You can use this action with optional inputs to generate various test and coverage artifacts.
 
-### Run MATLAB Tests on Self-Hosted Runner
-Use a [self-hosted runner](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) to run the tests in your [MATLAB project](https://www.mathworks.com/help/matlab/projects.html).
+### Run Tests in MATLAB Project
```

Comment:
For some reason, it seems weird that the first example we show is for how to use a self-hosted runner. I guess this has always been the case. I wonder how many people will try and copy/paste without thinking much about it if we take out the "on Self-Hosted Runner".

Reply:
It looks like Mark had the same comment in a different action. I agree with him that it's probably outside of the scope of the PR, and we can see what kind of feedback we get, but it's something to consider.

Reply:
Yes, we can see how it goes. For now, I prefer to keep the self-hosted examples at the top as they have one less action (less code). Each page has at most one self-hosted example and the majority of examples are GitHub-hosted.